test/test_S3Loader.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import time
+import unittest
+
+import dbinfo
+import numpy  as np
+
+from aperturedb import Connector, Status
+from aperturedb import ImageLoader
+
+class TestLoader(unittest.TestCase):
```

Comment:
this is the same as test_Loader.py. It would be better to simply import test_Loader.py here.
Or, you can just move the class TestS3Loader to test_Loader.py. S3 loader is yet another type of loader.